.eslintrc

```diff
@@ -13,6 +18,97 @@
         "exports": "always-multiline",
         "functions": "never"
       }
-    ]
+    ],
+
+    /** no-unsafe-innerhtml rule **/
+    "no-unsafe-innerhtml/no-unsafe-innerhtml" : 2,
+
+    /** ScanJS rules **/
```

Comment:
From the Before You Ship guide: https://before-you-ship.18f.gov/security/static-analysis/#javascript